interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/video-control-bar.twig

```diff
@@ -1,17 +1,24 @@
 <div class="telehealth-button-bar">
-    <button type="button" class="btn btn-default telehealth-btn telehealth-btn-notes d-none" aria-label="Toggle Notes" >
+    <button type="button" class="btn btn-default telehealth-btn telehealth-btn-notes d-none" aria-label="{{ "Toggle Notes"|xlt}}" title="{{ "Toggle Notes"|xlt}}" >
         <i class="fa fa-notes-medical fa-lg text-primary"></i>
```

Comment:
xla times 2 above line and all edited lines below

Reply:
Fixed